drupal8-apache/7.0/usr/local/share/container/baseimage-30.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+alias_function do_build do_drupal8_build_inner
```

Comment:
Is it possible to reverse the naming of the aliases?

I understand the reasoning of renaming to be `do_drupal8_build_inner` is so a child image down the line can compose a do_build function that calls upon different components.

What about `alias_function do_build do_php_apache_build`?
It would still allow the composition approach in child images but would clearly indicate where the original function came from, based on what parent image was above this.